schedule/jeos/sle/rpi/jeos-main.yaml

```diff
@@ -2,8 +2,8 @@
 description: 'Main RPi test suite. Maintainer: dheidler.'
 name: 'jeos-main@aarch64 ( qemu )'
 schedule:
-  - jeos/image_info
   - jeos/firstrun
+  - jeos/image_info
```

Comment:
Question is how much the `firstrun` step causes the HDD to grow. Can you check whether that's significant?

Reply:
I'm getting the image size from the worker pool directory, e.g. https://openqa.opensuse.org/tests/2337465#step/image_info/2
 `-rw------- 2 _openqa-worker nogroup 226623488 May 10 07:36 openSUSE-Tumbleweed-Minimal-VM.x86_64-1.0.0-kvm-and-xen-Snapshot20220509.qcow2`
 
 But this file is used to boot the VM directly,
 `[2022-05-10T07:36:22.359786+02:00] [debug] running `/usr/bin/qemu-img create -f qcow2 -F qcow2 -b /var/lib/openqa/pool/11/openSUSE-Tumbleweed-Minimal-VM.x86_64-1.0.0-kvm-and-xen-Snapshot20220509.qcow2 /var/lib/openqa/pool/11/raid/hd0-overlay0 25769803776`
 
so it doesn't really matter, this qcow2 won't change.

Reply:
To clarify, you mean `isn't used to boot the VM directly`?

Reply:
yes, sorry. `it isn't used to boot` is the right sentence :)